zenodo/modules/deposit/utils.py

```diff
@@ -265,7 +265,10 @@ def is_user_verified():
     """Permission function that evaluates if the user can create a deposit."""
     if current_user.email:
         email_domain = current_user.email.rsplit('@', 1)[-1].lower()
-        if current_domain_forbidden_list.is_forbidden(email_domain):
+        forbiddenlist = DomainList(current_app.config.get(
```

Comment:
We need this to be initialized in `ext.py` and be exposed via the proxy, since then the file is only loaded once (at application startup), and not every time this check happens.